### PR TITLE
{bio,tools}[foss/2023b] GROMACS v2024.1, networkx v3.2.1

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso
CORE_CNT=16

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen3
CORE_CNT=16

**Review by ocaisa — APPROVED** [review]
LGTM I also see a lot of successful builds in EESSI (though Haswell seems to have failed...)

File: easybuild/easyconfigs/g/GROMACS/GROMACS-2024.1-foss-2023b.eb [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+exts_default_options = {
+    'source_urls': [PYPI_SOURCE],
+    'use_pip': True,
+    'download_dep_fail': True,
+    'sanity_pip_check': True,
+}
+
+exts_list = [
+    ('gmxapi', '0.4.2', {
+        'preinstallopts': 'export CMAKE_ARGS="-Dgmxapi_ROOT=%(installdir)s -C %(installdir)s/share/cmake/gromacs_mpi/gromacs-hints_mpi.cmake" && ',
```
Comment (ocaisa) [inline easybuild/easyconfigs/g/GROMACS/GROMACS-2024.1-foss-2023b.eb:75]: CI is complaining about this line length

**Comment by ocaisa** [comment]
@boegelbot please test @ generoso
CORE_CNT=16

**Comment by ocaisa** [comment]
@boegelbot please test @ jsc-zen3
CORE_CNT=16

**Comment by ocaisa** [comment]
This looks ready to merge, just waiting on [some test builds from EESSI](https://github.com/EESSI/software-layer/pull/499)

**Review by ocaisa — APPROVED** [review]
LGTM

**Comment by ocaisa** [comment]
Haswell succeeded on EESSI with the workaround for #19940 (the [full fix is in the works](https://github.com/EESSI/software-layer/pull/488)), last box to tick is a [build for Neoverse V1](https://github.com/EESSI/software-layer/pull/499#issuecomment-1994588617)

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 21 out of 21 (2 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/verdurin/42f7149e769dd47d70eb6d749492a03d for a full test report.

**Comment by verdurin** [comment]
@ocaisa will your ARM build result arrive today, so I can merge this?

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 3 out of 3 (2 easyconfigs in total)
bear-pg0105u03a - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/branfosj/93ac011c284aace0649973ecb9d3c2c8 for a full test report.

**Comment by branfosj** [comment]
Going in, thanks @boegel!

**Pull request merged**